### PR TITLE
[MU4] Porting 3.5.2_backend fixes to master

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

**Review by RomanPudashkin — APPROVED** [review]

File: libmscore/synthesizerstate.h [hunk 1]
```diff
@@ -86,8 +86,14 @@ class SynthesizerState : public std::list<SynthesizerGroup>
...
+          {
+              0, "Zita1"
+          },
+#endif
+          {
+              2, "0.1"
+          },
           { 3, "440" },
           { 4, "1" },
           { 5, "1" }
```
Comment (Howard-C) [inline libmscore/synthesizerstate.h:99]: Why only change the first two and not all others?

Comment (Jojo-Schmitz) [reply libmscore/synthesizerstate.h:99]: Because the code style check only wanted those. IMHO bad enough...

Comment (Howard-C) [reply libmscore/synthesizerstate.h:99]: This is the problem of introducing automatic check. Do we still get to vote for its expulsion from MuseScore? :)

Comment (Jojo-Schmitz) [reply libmscore/synthesizerstate.h:99]: It comes with a price tag for sure...

**Pull request merged**